### PR TITLE
Us 31 tn 74: Removes hardcoding for the velocity graph api integration

### PR DESCRIPTION
**Pull request by AshankDsouza**

### CODE REVIEW

File: client/actions/project.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export async function getVelocity(projectSlug: string, sprintId: string) {
         headers: myHeaders
     };
     //http://localhost:8080/api/getDataForLeadTime?projectSlug=ser516asu-ser516-team-kansas&sprintNo=1
```
Comment (psing161) [inline client/actions/project.ts]: remove the commented line

**Review by psing161 — COMMENTED** [review]
Put some minor comments

**Comment by AshankDsouza** [comment]
Please review again

**Review by psing161 — APPROVED** [review]
LGTM

**Pull request merged**